test/jdk/ProblemList.txt

```diff
@@ -874,7 +874,7 @@ java/awt/Modal/PrintDialogsTest/PrintDialogsTest.java 8068378 generic-all
 java/awt/dnd/DnDFileGroupDescriptor/DnDFileGroupDescriptor.html 8080185 macosx-all,linux-all
 javax/swing/JTabbedPane/4666224/bug4666224.html 8144124  macosx-all
 java/awt/event/MouseEvent/AltGraphModifierTest/AltGraphModifierTest.java 8162380 generic-all
-java/awt/Window/TranslucentJAppletTest/TranslucentJAppletTest.java 8163086 macosx-all
+java/awt/Window/TranslucentJAppletTest/TranslucentJAppletTest.java 8163086 macosx-all,linux-all
```

Comment:
The failure of this test on Linux was reported on 02.2019 last time.  Since then only the problem on macOS is reported. Does the test actually fail on Linux right now?